header/sync/sync.go

```diff
@@ -119,35 +133,6 @@ func (s *Syncer) State() State {
 	return state
 }
 
-// trustedHead returns the latest known trusted header that is within the trusting period.
-func (s *Syncer) trustedHead(ctx context.Context) (*header.ExtendedHeader, error) {
-	// check pending for trusted header and return it if applicable
-	// NOTE: Pending cannot be expired, guaranteed
-	pendHead := s.pending.Head()
-	if pendHead != nil {
-		return pendHead, nil
-	}
-
-	sbj, err := s.store.Head(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	// check if our subjective header is not expired and use it
-	if !sbj.IsExpired() {
-		return sbj, nil
-	}
-
-	// otherwise, request head from a trustedPeer or, in other words, do automatic subjective initialization
-	objHead, err := s.exchange.Head(ctx)
-	if err != nil {
-		return nil, err
-	}
-
-	s.pending.Add(objHead)
-	return objHead, nil
```

Comment:
If we change from objective header to network header, we should probably also change the local var naming

Reply:
You pointed to removed code. In new code everything is renamed

Reply:
Oh, ok sorry! Was shown the new file, so assumed the other files were also synced.. weird